libqtile/core/manager.py

```diff
@@ -405,25 +405,35 @@ def reconfigure_screens(self, ev: Any = None) -> None:
     def paint_screen(self, screen: Screen, image_path: str, mode: str | None = None) -> None:
         self.core.painter.paint(screen, image_path, mode)
 
-    def process_key_event(self, keysym: int, mask: int) -> None:
+    def process_key_event(
+        self, keysym: int, mask: int
+    ) -> Tuple[Optional[Union[Key, KeyChord]], bool]:
```

Comment:
We no longer need Optional, Tuple or Union:
```suggestion
    ) -> tuple[Key | KeyChord | None, bool]:
```

Reply:
Ah yes thanks, that is really neat